cli.js

```diff
@@ -175,7 +175,7 @@ cli.main(function(args, options) {
     if (minified !== null) {
       // Write the output
       try {
-        if (output !== null) {
+        if (output !== null && output !== undefined) {
```

Comment:
I think jshint ignores sloppy equal comparison when it's against `null` for this exact reason